### PR TITLE
Fix line highlighting issues

### LINKED ISSUE #2: Highlighting breaks with camelCased file names
Hi @weirdpattern, love your plugin, been using it a lot [on my site lately](https://tyhopp.com).

It looks like the line highlighting breaks with file names that contain capitals, because the id applied to lines becomes all lowercase.

I'll submit a PR with a proposed fix 👌

### LINKED ISSUE #4: Highlighting regex is not global
Also found that the regex [here](https://github.com/weirdpattern/gatsby-remark-embed-gist/blob/f21b93c575f0080a07e21978aee05db3d082047e/src/index.js#L132) wasn't changing a file name such as file.js to file-js for line highlighting.

### PR DESCRIPTION
**Pull request by tyhopp**
Fixes #2 by converting file names to lowercase before applying highlighted id to specified lines.
Fixes #4 by making regex global.

### CODE REVIEW

**Comment by tyhopp** [comment]
@weirdpattern do you know if this would automatically update the plugin on the Gatsby site or do they maintain a fork of this repo?

File: src/index.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default async ({ markdownAST }, options = {}) => {
       const $ = cheerio.load(content.div);
       const file = query.file.replace(/[^a-zA-Z0-9_]+/, "-");
       query.highlights.forEach(line => {
-        $(`#file-${file}-LC${line}`).addClass("highlighted");
+        $(`#file-${file.toLowerCase()}-LC${line}`).addClass("highlighted");
```
Comment (weirdpattern) [inline src/index.js]: Let's optimize this a bit, shall we?
Move the `.toLowerCase()` call outside the loop.

Line 132 seems like a good place to do this...
```js
const file = query.file.replace(/[^a-zA-Z0-9_]+/g, "-").toLowerCase();
```

Comment (tyhopp) [reply src/index.js]: Looks good, you're right that is a better place for it 🙂

**Comment by tyhopp** [comment]
Changed to suggested placement ✅

**Pull request merged**